src/main/java/seedu/address/logic/parser/UntagCommandParser.java

```diff
@@ -15,42 +20,66 @@
  */
 public class UntagCommandParser implements Parser<UntagCommand> {
     public static final int MAX_LENGTH = 50;
+    public static final int MAX_INDEXES = 10;
```

Comment:
Could this be placed in the Index class for neatness?

Reply:
Good catch!